### DISCUSSION TITLE
Import Payments

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#94] [General]
On the Approved Bookings page there's an option to: “Click on the icon to show the import form to import payments”, but I can't find any documentation for the fields required and accepted. Can you please let me know? And, forgive me if I've overlooked something in the documentation or forums. Thanks in advance.

**Comment by liedekef** [comment]
This is now documented: https://www.e-dynamics.be/wordpress/csv-import-payments/

**Reply by roysinn** [reply]
Thanks again Franky.
